src/swagger-specs/segmentation.yaml

```diff
@@ -2988,16 +2960,20 @@ components:
               - $ref: '#/components/schemas/ExternallyGeneratedAudience'
         _page:
           type: object
+          description: An object that shows what page the list is on.
           properties:
             totalCount:
               type: integer
               example: 111
+              description: The total number of audiences.
             pageSize:
               type: integer
               example: 2
+              description: The size of the audience page.
             next:
               type: integer
               example: 1
+              description: The value needed for `start` to show the next page of results.
         _links:
           type: object
```

Comment:
`_links` needs a description, even if it just contains a single double-nested property. It's a bit repetitive, but unfortunately Redocly always collapses to the top-level properties and we want to avoid blank descriptions.